docs/pai-job-protocol.yaml

```diff
@@ -19,16 +19,26 @@ prerequisites: # Optional
     description: String, optional
     auth: Object, optional # Only available when the type is dockerimage.
       username: String, optional
-      passward: String, optional
+      password: String, optional # If a password is needed, it should be referenced as a secret
       registryuri: String, optional
     uri: String or list, required # Only when the type is data can the uri be a list.
 
 # If specified, the whole parameters object can be referenced as `$parameters`.
-# Scope of reference `$parameters`: the refercen is global and shared among all task roles.
+# Scope of reference `$parameters`: the reference is global and shared among all task roles.
 parameters: # Optional, can be omitted.
   <param1>: value1 # Specify name and value of all the referencable parameters that will be used in the whole job template.
   <param2>: value2 # Can be referenced by `<% $parameters.param1 %>`, `<% $parameters.param2 %>`.
 
+# If sensitive information including password or API key is needed in the protocol,
+# it should be specified here in secrets section and referenced as `$secrets`.
+# Scope of reference `$secrets`: the reference is global, shared among all task roles and docker image's `auth` field.
```

Comment:
"global" means all sections. I think the reference scope is taskRole and auth field.

Reply:
updated